assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -347,13 +347,6 @@ const Checkout = ( { attributes, scrollToTop } ) => {
 										}
 									/>
 								) }
-								{ /*@todo This is not implemented*/ }
-								<CheckboxControl
-									className="wc-block-checkout__add-note"
```

Comment:
Should we remove the [CSS selector](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/blocks/cart-checkout/checkout/style.scss#L5) targeting this checkbox?

Reply:
We can keep it; I think I'll give adding this back a shot during cooldown once all other issues are resolved :) Ref #1483